Tell Chromium to look in /etc/chromium for the master_preferences file
--- a/src/chrome/common/chrome_paths.cc
+++ b/src/chrome/common/chrome_paths.cc
@@ -307,7 +307,7 @@ bool PathProvider(int key, FilePath* res
 #if defined(GOOGLE_CHROME_BUILD)
       cur = FilePath(FILE_PATH_LITERAL("/etc/opt/chrome/policies"));
 #else
-      cur = FilePath(FILE_PATH_LITERAL("/etc/chromium/policies"));
+      cur = FilePath(FILE_PATH_LITERAL("/etc/chromium-browser/master_preferences"));
 #endif
       if (!file_util::PathExists(cur))  // we don't want to create this
         return false;
--- a/src/chrome/browser/first_run/first_run.cc
+++ b/src/chrome/browser/first_run/first_run.cc
@@ -93,9 +93,7 @@ bool FirstRun::ProcessMasterPreferences(
 
   // The standard location of the master prefs is next to the chrome binary.
   FilePath master_prefs;
-  if (!PathService::Get(base::DIR_EXE, &master_prefs))
-    return true;
-  master_prefs = master_prefs.AppendASCII(installer::kDefaultMasterPrefs);
+  master_prefs = FilePath("/etc/chromium");
 
   installer::MasterPreferences prefs(master_prefs);
   if (!prefs.read_from_file())
